### DISCUSSION TITLE
HT3D error intepretation

### DISCUSSION THREAD

**Discussion by R-Godakandage** [firemodels/fds#13796] [Q&A] [answered]
Could you please assist me in interpreting the following error. 

ERROR(423): HT3D solid must have at least  one face exposed in direction 2 : Mesh 23, IOR= -1, IIG=2,JJG=12, KKG=1, I=1.

Thanks a lot.

**Comment by drjfloyd** [comment, marked as answer]
Open the User's Guide. In the back is a list of all error codes. Find error code 423. It will point to a section in the Guide. Read that Section.

**Reply by R-Godakandage** [reply]
Thanks a lot . I got it.
